brainstormforce/ultimate-addons-for-gutenberg
# SPCTR-609 Fix: FAQ - Resolved row and column gap not working in editor.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/c6482899-7e6a-4642-8172-f84e58f836b4?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots

- https://share.getcloudapp.com/kpuJjBDO

### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop the FAQ block.
- Check Accordion layout Row Gap in editor and front.
- Check Grid layout Column and Row gap in the editor as well as on frontend.

### Checklist:
